lib/one_time_password.rb

```diff
@@ -10,11 +10,11 @@ module OneTimePassword
   private
 
   def generate_otp
-    ROTP::TOTP.new(OTP_SECRET, issuer: "Claim Additional Payments for Teaching", interval: OTP_PASSWORD_INTERVAL).now
+    ROTP::TOTP.new(OTP_SECRET, issuer: "Claim Additional Payments for Teaching").now
   end
 
   def verify_otp(challenge_code)
-    totp = ROTP::TOTP.new(OTP_SECRET, issuer: "Claim Additional Payments for Teaching", interval: OTP_PASSWORD_INTERVAL)
-    totp.verify(challenge_code)
+    totp = ROTP::TOTP.new(OTP_SECRET, issuer: "Claim Additional Payments for Teaching")
+    totp.verify(challenge_code, drift_behind: OTP_PASSWORD_INTERVAL)
```

Comment:
@d-a-v-e So if I understand correctly this will be our 900 seconds (15 minutes) + whatever seconds remaining on the default of 30 seconds? So min would be 901 seconds (15 mins 1 second) and maximum would be 929 seconds (15 mins 29 seconds)?

Reply:
sounds about right!